### PR TITLE
Improve installation doc

### PR DESCRIPTION
**Pull request by mwestphal**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -436,6 +476,15 @@ Be sure that VTK has been built with *OpenImageDenoise* support (`VTKOSPRAY_ENAB
   * If no formats have working thumbnails, then it is an issue with the f3d.thumbnailer file
 
 ## Windows
+> After installing F3D or registering the shell extension, my explorer is broken
+
+Unregister the shell extension by running:
+
+```
+cd C:\path\to\f3d\bin\
+regsvr32 /u F3DShellExtension.dll
```
Comment (mwestphal) [inline README.md:485]: did you try this to fix the issue ? @Meakk

File: README.md [hunk 2]
```diff
@@ -385,17 +385,57 @@ On Linux, it will be installed in `/etc/f3d/`, on Windows, it will be installed
...
+To remove the shell extension, run:
+
+```
+cd C:\path\to\f3d\bin\
+regsvr32 /u F3DShellExtension.dll
+```
+
+## MacOS
+
+There is no support for thumbnails on MacOS, both the [brew](##Brew) package and .dmg binary release should provide automatic file openings.
```
Comment (mwestphal) [inline README.md:438]: is this correct @Meakk ?

**Comment by mwestphal** [comment]
@drtrigon please take a look

File: README.md [hunk 3]
```diff
@@ -385,17 +385,57 @@ On Linux, it will be installed in `/etc/f3d/`, on Windows, it will be installed
...
+1. Extract F3D archive in a TEMP folder
+2. move $TEMP/config.json to ~/.config/f3d/
+3. copy $TEMP/* to ~/.local/
+4. Update your [mime database](https://linux.die.net/man/1/update-mime-database) pointing to ~/.local/share/mime
+5. Update your [desktop database](update-desktop-database](https://linuxcommandlibrary.com/man/update-desktop-database) pointing to ~/.local/share/application
+
+```bash
+tar -xzvf f3d-1.2.0-Linux.tar.gz
+cd f3d-1.2.0-Linux
+mkdir ~/.config/f3d/
```
Comment (Meakk) [inline README.md:407]: mkdir -p

File: README.md (hunk 2 above)
Comment (Meakk) [reply README.md:438]: I don't think it's true for brew

File: README.md (hunk 1 above)
Comment (Meakk) [reply README.md:485]: Nop I had to use ShellExView to disable it

**Review by Meakk — CHANGES_REQUESTED** [review]

**Review by Meakk — APPROVED** [review]

**Comment by drtrigon** [comment]
@mwestphal: Sorry for answering that late ... looks good now! Thanks!

**Pull request merged**